src/Forms/GridField/GridFieldPrintButton.php

```diff
@@ -126,21 +126,26 @@ public function getURLHandlers($gridField)
      */
     public function handlePrint($gridField, $request = null)
     {
+        $orig = ini_get('max_execution_time');
         set_time_limit(60);
```

Comment:
It's not even clear why this limit is here, it's part of the initial commit that adds this component and has no commentary to go with it. We might want to remove it in the future. For now I agree to leaving it here, since this is a patch and we don't yet know what the ramifications of removing it might be.